STYLEGUIDE.md

```diff
@@ -1,39 +1,311 @@
+
+> When he logs out using the button, he will be taken to the login page.
+
+### 3.4 Avoid "click"
+
+"Click" is an outdated term that assumes the learner is using a mouse. But learners may be navigating by touchscreen, keyboard, or assistive technology. Often there are better words than "click"—like "select" and "open".
+
+#### Do
+
+> Open the **Settings tab.**
+
+> Select the **Google** social connection.
+
+#### Don't
+
+> Click the **Settings tab.** 
+
+> Click the **Google** social connection.
+
+## 4. Write for code
+
+### 4.1 Use monospace fonts for code, commands, file names, copy-ables, and URLs; use bold for menus, and other UI copy.
+
+#### Do
+
+> In your [Clerk Dashboard](https://dashboard.clerk.com/), go to **User & Authentication > Social Connections** and open the **Settings tab.**
+
+> In your browser, open [`http://localhost:3000/`](http://localhost:3000/).
+
+#### Don't
+
+> In your [Clerk Dashboard](https://dashboard.clerk.com/), go to `User & Authentication > Social Connections` and open the "Settings tab."
+
+> In your browser, open [`http://localhost:3000/`](http://localhost:3000/).
+
+### 4.2 Use carets to help users navigate menus; 
+
+Use carets to nest operations of the same type like menu navigation.
+
+#### Do
+
+> In your [Clerk Dashboard](https://dashboard.clerk.com/), go to **User & Authentication > Social Connections** and open the **Settings tab.**
+
+#### Don't
+
+> In your [Clerk Dashboard](https://dashboard.clerk.com/), go to **User & Authentication**, click on **Social Connections**, and open the **Settings tab.**
+
+
+### 4.3 Write compliant JSON: no single quotes, no comments
+
+Conform to the [JSON standard](https://www.json.org/json-en.html):
+
+* Key names are enclosed in double quotes (`"`)
+* Values are between double quotes (`"`), not single quotes (`'`)
+* No comments, neither `//` nor `/* */`
+
+#### Do
+
+> This is the user's primary email address in the User object
+
+```json
+{
+  "email": "{{user.primary_email_address}}"
+}
+```
+
+#### Don't
+
+```json
+{
+	// this is the user's primary email address in the User object
+  email: '{{user.primary_email_address}}'
+}
+```
+ 
+### 4.4 Use 2-space indents for code blocks
+
+We use 2-space indents to conserve horizontal space in code blocks.
+
+#### Do
+
+```js
+export const config = {
+  matcher: ['/((?!.+\\.[\\w]+$|_next).*)', '/', '/(api|trpc)(.*)'],
+};
+```
+
+#### Don't
+
+```js
+export const config = {
+    matcher: ['/((?!.+\\.[\\w]+$|_next).*)', '/', '/(api|trpc)(.*)'],
+};
+```
+
+### 4.5 Authentication states should be hyphenated, bold, and use monospace fonts.
+
+Clerk authentication states **`signed-in`**, **`signed-out`**, and **`unknown`** should be hyphenated, bold, and use monospace fonts to distinguish them from general states of being.
+
+#### Do
+
+> Once you have a **`signed-in`** user, you need to give them a way to sign out.
+
+#### Don't
+
+> Once you have a signed-in user, you need to give them a way to sign out.
```

Comment:
I don't remember this being a discussion, and I truly believe this is going to create a lot of visual noise in the docs. Code snippets should be saved for code-related copy. Saying a user is signed-in is not necessarily code-related. I think if we want to bold the term, maybe that could be a feasible solution. I do not support using bold AND code snippets, nor code snippets at all, to distinguish signed-in, signed-up, signed-out states.

Reply:
I think I side with Alexis on this. I feel like bold is a good choice. Using backticks could feel out of place and confusing to developers reading the docs.

Reply:
Bold it is! Thanks for reviewing and adding feedback!